### ISSUE TITLE
segmentation fault running testing environment

### ISSUE THREAD

**Issue by nholman** [qutip/qutip#1124]
I am trying to learn to use qutip and I made a local environment. As per the instruction guide, I ran the qutip testing environment and got the following: 

```
QuTiP: Quantum Toolbox in Python
================================
Copyright (c) QuTiP team 2011 and later.
Original developers: R. J. Johansson & P. D. Nation.
Current admin team: Alexander Pitchford, Paul D. Nation, Nathan Shammah, Shahnawaz Ahmed, Neill Lambert, and Eric Giguère.
Project Manager: Franco Nori.
Currently developed through wide collaboration. See https://github.com/qutip for details.

QuTiP Version:      4.4.1
Numpy Version:      1.17.3
Scipy Version:      1.3.1
Cython Version:     0.29.13
Matplotlib Version: 3.1.1
Python Version:     3.7.3
Number of CPUs:     4
BLAS Info:          Generic
OPENMP Installed:   False
INTEL MKL Ext:      False
Platform Info:      Darwin (x86_64)
Installation path:  /Users/nathanholman/opt/anaconda3/envs/qutip/lib/python3.7/site-packages/qutip
==============================================================================
Please cite QuTiP in your publication.
==============================================================================
For your convenience a bibtex reference can be easily generated using `qutip.cite()`
Transform 2-level to eigenbasis and back ... ok
Transform 10-level real-values to eigenbasis and back ... ok
Transform 10-level to eigenbasis and back ... ok
Transform 10-level imag to eigenbasis and back ... ok
Consistency between transformations of kets and density matrices ... ok
Check diagonalization via eigenbasis transformation ... ok
Check Qobj eigs and direct eig solver transformations match ... ok
Check Qobj eigs and direct eig solver reverse transformations match ... ok
brmesolve: simple qubit ... ok
brmesolve: c_ops alone ... ok
brmesolve: c_ops with a_ops ... ok
brmesolve: harmonic oscillator, zero temperature ... ok
brmesolve: harmonic oscillator, finite temperature ... ok
brmesolve: harmonic oscillator, finite temperature, states ... ok
brmesolve: Jaynes-Cummings model, zero temperature ... ok
brmesolve: Check for #572 bug. ... ok
brmesolve: input list of Qobj ... ok
td_brmesolve: passes all brmesolve tests ... ok
td_brmesolve: time-dependent a_ops ... ok
td_brmesolve: time-dependent a_ops tuple of strings ... ok
td_brmesolve: time-dependent a_ops tuple interp ... ok
td_brmesolve: time-dependent a_ops & c_ops interp ... ok
td_brmesolve: non-Hermitian e_ops check ... ok
td_brmesolve: states check ... ok
td_brmesolve: split ops #1 ... ok
td_brmesolve: split ops #2 ... ok
td_brmesolve: split ops, Cubic_Spline td-terms ... ok
td_brmesolve: split ops, multiple ... ok
td_brmesolve: Hamiltonian args ... ok
BR Tools : zheevr ... Segmentation fault: 11
```

I'm unsure what is going on here. I tried reinstalling qutip without any change. Prior I had a lot of errors referring to xcrun but I fixed that by installing xcode. Is there something else I need to get this working?

**Comment by WBCardoso** [comment]
I am with the same problem. Note that the QuTiP version and the python version is different. I have installed the python via macports, but I have also tested in anaconda system. The problem is the same in the anaconda.

```
QuTiP: Quantum Toolbox in Python
================================
Copyright (c) QuTiP team 2011 and later.
Original developers: R. J. Johansson & P. D. Nation.
Current admin team: Alexander Pitchford, Paul D. Nation, Nathan Shammah, Shahnawaz Ahmed, Neill Lambert, and Eric Giguère.
Project Manager: Franco Nori.
Currently developed through wide collaboration. See https://github.com/qutip for details.

QuTiP Version:      4.5.0.dev0+3421bc5c
Numpy Version:      1.17.4
Scipy Version:      1.3.2
Cython Version:     0.29.14
Matplotlib Version: 3.1.1
Python Version:     3.8.0
Number of CPUs:     4
BLAS Info:          OPENBLAS
OPENMP Installed:   False
INTEL MKL Ext:      False
Platform Info:      Darwin (x86_64)
Installation path:  /opt/local/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/qutip-4.5.0.dev0+3421bc5c-py3.8-macosx-10.15-x86_64.egg/qutip
==============================================================================
Please cite QuTiP in your publication.
==============================================================================
For your convenience a bibtex reference can be easily generated using `qutip.cite()`
DEBUG:qutip.logging_utils:Creating logger for qutip.control.grape with policy basic.
DEBUG:qutip.logging_utils:Creating logger for qutip.control.optimconfig with policy basic.
DEBUG:qutip.logging_utils:Creating logger for qutip.control.dynamics with policy basic.
DEBUG:qutip.logging_utils:Creating logger for qutip.control.dump with policy basic.
DEBUG:qutip.logging_utils:Creating logger for qutip.control.tslotcomp with policy basic.
DEBUG:qutip.logging_utils:Creating logger for qutip.control.fidcomp with policy basic.
DEBUG:qutip.logging_utils:Creating logger for qutip.control.propcomp with policy basic.
DEBUG:qutip.logging_utils:Creating logger for qutip.control.optimizer with policy basic.
DEBUG:qutip.logging_utils:Creating logger for qutip.control.pulsegen with policy basic.
DEBUG:qutip.logging_utils:Creating logger for qutip.control.loadparams with policy basic.
DEBUG:qutip.logging_utils:Creating logger for qutip.control.pulseoptim with policy basic.
Transform 2-level to eigenbasis and back ... ok
Transform 10-level real-values to eigenbasis and back ... ok
Transform 10-level to eigenbasis and back ... ok
Transform 10-level imag to eigenbasis and back ... ok
Consistency between transformations of kets and density matrices ... ok
Check diagonalization via eigenbasis transformation ... ok
Check Qobj eigs and direct eig solver transformations match ... ok
Check Qobj eigs and direct eig solver reverse transformations match ... ok
brmesolve: simple qubit ... ok
brmesolve: c_ops alone ... ok
brmesolve: c_ops with a_ops ... ok
brmesolve: harmonic oscillator, zero temperature ... ok
brmesolve: harmonic oscillator, finite temperature ... ok
brmesolve: harmonic oscillator, finite temperature, states ... ok
brmesolve: Jaynes-Cummings model, zero temperature ... ok
brmesolve: Check for #572 bug. ... ok
brmesolve: input list of Qobj ... ok
td_brmesolve: passes all brmesolve tests ... ok
td_brmesolve: time-dependent a_ops ... ok
td_brmesolve: time-dependent a_ops tuple of strings ... ok
td_brmesolve: time-dependent a_ops tuple interp ... ok
td_brmesolve: time-dependent a_ops & c_ops interp ... ok
td_brmesolve: non-Hermitian e_ops check ... ok
td_brmesolve: states check ... ok
td_brmesolve: split ops #1 ... ok
td_brmesolve: split ops #2 ... ok
td_brmesolve: split ops, Cubic_Spline td-terms ... ok
td_brmesolve: split ops, multiple ... ok
td_brmesolve: Hamiltonian args ... ok
BR Tools : zheevr ... 
```

**Comment by goerz** [comment]
Probable duplicate of #1197

**Comment by jakelishman** [comment]
This particular segfault was fixed by #1288.
